src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,620 @@
+# -*- coding: latin-1 -*-
+from collections import OrderedDict
+
+
```

Comment:
In general we need to re-work the fundamental configuration data structures, but initial focus needs to be on covering the current code with tests.

Reply:
@dnwillia-work @seanpearsonuk @h-krishnan @mkundu1 

Now we support all temperature conversions as well.  

We will re-work on data structures and now focused on covering all possible testing.

Thank you.